### PR TITLE
Change METRIC_ACCUMULATION_LEVEL to a list of multiple values

### PR DESCRIPTION
**Pull request by bgruening**
fixes: https://github.com/galaxyproject/tools-devteam/issues/244

### CODE REVIEW

File: tools/picard/picard_CollectAlignmentSummaryMetrics.xml [hunk 1]
```diff
@@ -24,7 +24,9 @@
     #for $sequence in $adapters:
         ADAPTER_SEQUENCE="${sequence.adapter}"
     #end for
-    METRIC_ACCUMULATION_LEVEL="${metric_accumulation_level}"
+    #for $level in ${metric_accumulation_level}:
```
Comment (scholtalbers) [inline tools/picard/picard_CollectAlignmentSummaryMetrics.xml]: Hi @bgruening, this does not work for me (galaxy@15.07). 
*Long-form placeholders - ${}, $(), $[], etc. are not valid inside expressions.* ==> should be ``in $metric_accumulation_level``. 
But changing that results in:
``TypeError: 'SelectToolParameterWrapper' object is not iterable``

**Comment by bgruening** [comment]
Tests are passing for me now.

**Comment by scholtalbers** [comment]
Yes, this works, thx!

**Comment by martenson** [comment]
updated on MTS

**Pull request merged**